[2025-08-28T13:01:04.411+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-08-28T13:01:04.414+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: march_madness_predictions.predict manual__2025-08-28T13:00:08+00:00 [queued]>
[2025-08-28T13:01:04.416+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: march_madness_predictions.predict manual__2025-08-28T13:00:08+00:00 [queued]>
[2025-08-28T13:01:04.416+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-08-28T13:01:04.420+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): predict> on 2025-08-28 13:00:08+00:00
[2025-08-28T13:01:04.422+0000] {standard_task_runner.py:72} INFO - Started process 412 to run task
[2025-08-28T13:01:04.423+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'march_madness_predictions', 'predict', 'manual__2025-08-28T13:00:08+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/instructions.py', '--cfg-path', '/tmp/tmpdz49bgri']
[2025-08-28T13:01:04.424+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask predict
[2025-08-28T13:01:04.437+0000] {task_command.py:467} INFO - Running <TaskInstance: march_madness_predictions.predict manual__2025-08-28T13:00:08+00:00 [running]> on host b7e162002360
[2025-08-28T13:01:04.454+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='march_madness_predictions' AIRFLOW_CTX_TASK_ID='predict' AIRFLOW_CTX_EXECUTION_DATE='2025-08-28T13:00:08+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-28T13:00:08+00:00'
[2025-08-28T13:01:04.455+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-08-28T13:01:04.460+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-28T13:01:04.461+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/airflow/scripts/predict.py']
[2025-08-28T13:01:04.465+0000] {subprocess.py:86} INFO - Output:
[2025-08-28T13:01:05.049+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-28T13:01:05.056+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-08-28T13:01:05.056+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=march_madness_predictions, task_id=predict, run_id=manual__2025-08-28T13:00:08+00:00, execution_date=20250828T130008, start_date=20250828T130104, end_date=20250828T130105
[2025-08-28T13:01:05.081+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-08-28T13:01:05.086+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-28T13:01:05.086+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
